pkg/ctrlz/options.go

```diff
@@ -25,6 +25,9 @@ type Options struct {
 
 	// The IP address to listen on for ctrlz.
 	Address string
+
+	// Automatically pick a port instead of using the one in the Port field.
+	PickPort bool
```

Comment:
Can we follow the pattern at https://github.com/istio/istio/blob/9eef86e0d517149803ac58c8291515601fdc7857/galley/cmd/galley/cmd/root.go#L101-L103? This adds an explicit flag to enable/disable the feature, rather than rely on the zero value of the port.

Reply:
By feature do you mean [not] running CtrlZ server (so that we don't need PickPort and just use Port=0 to use any port)? If so, is there a reason to not simply skip running ctrlz if port is 0 (unless we want port picking to work from command line too)? So basically move the check up one level in the call stack.